.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,68 @@
+name: Bug Report
+description: Create a report to help us improve
+labels:
+  - "Status: Triage"
+  - "Type: Bug / Error"
+
+body:
+  - type: markdown
+    attributes:
+      value: |-
+        ## Before you submit...
+        We like to help you, but in order to do that should you make a few things first:
+        
+        - Use a clear and concise title
+        - Fill out the text fields with as much detail as possible.
+        - Never be shy to give us screenshots and/or code samples. It will help!
+  - type: textarea
+    attributes:
+      label: Description
+      description: Give a clear and concise description of what the bug is.
+      placeholder: When I do ... does ... happen.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: Give a step-by-step example on how to reproduce the bug.
+      placeholder: |-
+        1. Do this
+        2. Do that
+        3. ...
+        4. Bug!
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Screenshots
+      description: If applicable, add screenshots to help explain your issue.
+  - type: textarea
+    attributes:
+      label: Code Sample
+      description: |-
+        If applicable, add the code sample or a link to the [Live Editor](https://mermaid.live).
+        Any text pasted here will be rendered as a Code block.
+      render: text
```

Comment:
> Any text pasted here will be rendered as a Code block.

Change render (to mermaid)?

Or It'll be better to remove the `Any text pasted here...` message and let the user handle it with fences. That'll allow them to paste both JS & Mermaid code as required.

Reply:
I wanted it to be rendered like that, as it would allow you to easily see the raw code used here for the diagram.

I dislike the option of leaving it up to the user, as they for sure won't think about it in the end, leaving some large, messy text in the worst cases.